Sources/RTMP/RTMPStream.swift

```diff
@@ -389,6 +389,11 @@ open class RTMPStream: IOStream {
         }
     }
 
+    /// Sends a command to the RTMP client or server.
+    public func send(commandName: String, arguments: Any?...) {
```

Comment:
[IMO]
RTMPConneciton.call is already public, so I don't feel the need for implementation on the HaishinKit side. We expect it to be defined in the importing side's extension or RTMPStream extension.

Reply:
Sounds good. `connection` is private, but it's injected into the `RTMPStream` so it can be used from the outside.